src/test/java/io/strimzi/kafka/bridge/http/StaticTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.http;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class StaticTest {
+
+    public String getVersionFromFile() {
+
+        String versionFromFile = "Unable to get version";
+
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader("release.version"));
+            versionFromFile = bufferedReader.readLine();
+            bufferedReader.close();
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return versionFromFile;
+    }
+
+    @Test
+    void bridgeVersionDisplayedInStartupTest() throws IOException, InterruptedException {
+
+        String kBVersion = getVersionFromFile();
+        assertThat(kBVersion, is(not("Unable to get version")));
```

Comment:
Why not just throw some exception out of the `getVersionFromFile` method and let it fail the test? That would be IMHO more Java native solution than the `Unable to get version` text.